paasta_tools/graceful_app_drain.py

```diff
@@ -109,5 +94,27 @@ def main():
     paasta_print("Sucessfully killed {}".format(full_appid))
 
 
+def main():
+    args = parse_args()
+    full_appid = args.appname.lstrip('/')
+
+    system_paasta_config = load_system_paasta_config()
+    cluster = system_paasta_config.get_cluster()
+    clients = marathon_tools.get_list_of_marathon_clients(system_paasta_config=system_paasta_config)
+
+    for client in clients:
+        if marathon_tools.is_app_id_running(app_id=full_appid, client=client):
+            kill_marathon_app(
+                full_appid=full_appid,
+                cluster=cluster,
+                client=client,
+                soa_dir=args.soa_dir,
+            )
+            return 0
```

Comment:
This code assumes that it only exists on one shard. 

Instead can you move the return out of the loop and instead count how many apps were killed?
then return non-0 if apps_killed < 1

Reply:
Done.